setup.py

```diff
@@ -52,7 +52,7 @@ def remove_bdist_rpm_source_file():
 
 PACKAGE = {
     'name': 'vsc-base',
-    'version': '2.2.1',
+    'version': '2.2.2',
```

Comment:
is this really worth a version bump?

the `pass` should be there, for sure, but this is just a style cleanup (functionality was not changed)